README.md

```diff
@@ -91,6 +91,26 @@ Should be able to navigate to:
 
 > Note: Keycloak is required to run the app - [see below](https://github.com/hzi-braunschweig/SORMAS-Angular#auth-with-keycloak)
 
+
+## Docker Compose
+1. You will first need to add the following to your hosts file: `127.0.0.1	sormas-docker-test.com`
+2. On Windows ensure the local git repository is set to not replace line endings with
+   `git config core.autocrlf input` and ` git config core.eol lf`
+3. On Windows ensure your IDE is set to use LF for line endings instead of the default CRLF
+
+To run the environment please use: `docker-compose up`
+
+## Links:
+| App Name           | Url                                            |
+| -------------------| -----------------------------------------------|
+| Sormas Application | http://localhost:8100/sormas-ui/login          |
+| Sormas Angular     | http://localhost:8200                          |
+| Sormas Postgres    | 127.0.0.1:8300                                 |
+| Keycloak           | http://localhost:8400                          |
+| Keycloak Postgres  | 127.0.0.1:8401                                 |
+
```

Comment:
Maybe add the postgres port and give an example to access it with psql. `psql --host=localhost --port=8300 -U postgres sormas`. Might be super helpful to check if you enter something in the UI really ends up in the correct table :D

Reply:
Thanks for all your help @JonasCir! I think I will merge the PR as it's already very well documented. I use PGAdmin for Postgres stuff anyway :-)